app/views/layouts/mailer.text.erb

```diff
@@ -0,0 +1 @@
+<%= yield %>
```

Comment:
We may not need the views since it is an api.

Reply:
we choose to leave the views regardles

Reply:
No problem